src/Parser.js

```diff
@@ -179,7 +180,16 @@ module.exports = class Parser {
           out += top ? this.renderer.paragraph(body) : body;
           continue;
         }
+
         default: {
+          // Run any renderer extensions
+          tokenParsed = false;
+          if (this.options.extensions?.[token.type]) {
+            out += this.options.extensions[token.type](token);
+            tokenParsed = true;
+          }
+          if (tokenParsed) continue;
```

Comment:
```suggestion
          // Run any renderer extensions
          if (this.options.extensions?.[token.type]) {
            out += this.options.extensions[token.type](token);
            continue;
          }
```

Reply:
Good catch. Right, we shouldn't have multiple Renderers with the same `name`, so this is fine. That also might be something to make sure we clarify in the documentation.

This is fixed now.